### DISCUSSION TITLE
Server Error after upgrading to NetBox v3.4.0

### DISCUSSION THREAD

**Discussion by Arshnika** [netbox-community/netbox#11186] [Q&A] [answered]
Hi NetBoxers!

Today, we have upgraded NetBox to the latest major release v3.4.0. The upgrade was successfully completed. But, now, when I run scripts, I face the following Server Error but the scripts successfully run!

![image](https://user-images.githubusercontent.com/95662911/207770747-4fedcf52-8c6e-4309-84fc-11e55625f104.png)

Any idea how to rectify this?

Thanks.

**Comment by jeremystretch** [comment, marked as answer]
Looks like bug #11185

**Reply by Arshnika** [reply]
Hi Jeremy,

Thanks heaps for your comment. It really helped!

It seems that this bug will be fixed in release 3.4.1. The good news is the issue has been fixed by adding {% load humanize %} to the top of netbox/templates/extras/htmx/script_result.html file.

Really appreciate for your help.

Cheers,
